conda/recipes/libcuvs/build_libcuvs_tests.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 # Copyright (c) 2022-2024, NVIDIA CORPORATION.
 
-./build.sh tests bench-ann --allgpuarch --no-nvtx --build-metrics=tests_bench --incl-cache-stats
+./build.sh tests --allgpuarch --no-nvtx --build-metrics=tests_bench --incl-cache-stats
```

Comment:
Does the `--build-metrics` flag need to change if benchmarks are not being built here?
```suggestion
./build.sh tests --allgpuarch --no-nvtx --build-metrics=tests --incl-cache-stats
```